src/librbd/Operations.cc

```diff
@@ -1184,13 +1167,10 @@ int Operations<I>::snap_rename(const char *srcname, const char *dstname) {
     }
 
     r = cond_ctx.wait();
-    if (r < 0) {
-      return r;
-    }
   }
 
   m_image_ctx.perfcounter->inc(l_librbd_snap_rename);
```

Comment:
With this change, perf counter would now increment `l_librbd_snap_rename` even when rename wasn't successful or async snap remove request failed?  Is that okay?